src/react-components/src/components/03-organisms/SearchPromo/SearchPromo.tsx

```diff
@@ -27,7 +27,9 @@ export interface SearchPromoProps {
   textChangeHandler?: (event: React.FormEvent) => void;
 }
 export default function SearchPromo(props: SearchPromoProps) {
-  const { headingText, modifiers, blockName, titleId, searchBarId, searchButtonId, hasError, errorMessage,
+  const { headingText, modifiers, blockName, titleId,
```

Comment:
`modifiers` isn't being used in this file.

Reply:
So unlike the the other stuff I feel like `blockname`, `baseclass` and `modifiers` are stuff that are useful to have even if they're not used - mostly because it's /such a pain/ to get them all the way down the nested component if we ever need them in the future.  Does it hurt to have them in here?  Do unused variables get stripped out when it builds?  (Is there an easy way to copy a component all the way down the component tree?)